src/main/java/net/neoforged/neoforge/event/level/BlockEvent.java

```diff
@@ -99,6 +99,49 @@ public void setExpToDrop(int exp) {
         }
     }
 
+    /**
```

Comment:
New events should no longer use the innerclass pattern, we are currently in the process of moving away from these. You can still extend `BlockEvent`, but the event should be in its own file.

Reply:
Event has been moved into its own class under event/level.